package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "visma-e-commerce",
+  "version": "1.0.0",
+  "description": "",
+  "main": "gulpfile.js",
+  "dependencies": {
+    "array-sort": "^1.0.0",
+    "call-bind": "^1.0.2",
+    "error-ex": "^1.3.2",
+    "for-own": "^1.0.0",
+    "fs.realpath": "^1.0.0",
+    "function-bind": "^1.1.1",
+    "get-intrinsic": "^1.1.1",
+    "get-caller-file": "^1.0.3",
+    "get-value": "^2.0.6",
+    "graceful-fs": "^4.2.6",
+    "gulp-autoprefixer": "^7.0.1",
+    "gulp": "^4.0.2",
+    "gulp-babel": "^8.0.0",
+    "gulp-livereload": "^4.0.2",
+    "gulp-plumber": "^1.2.1",
+    "gulp-rename": "^2.0.0",
+    "gulp-sass": "^4.1.0",
+    "gulp-sourcemaps": "^3.0.0",
+    "gulp-uglify": "^3.0.2",
+    "has": "^1.0.3",
+    "has-symbols": "^1.0.2",
+    "hosted-git-info": "^2.8.9",
+    "http-server": "^0.12.3",
+    "inherits": "^2.0.4",
+    "ini": "^1.3.8",
+    "interpret": "^1.4.0",
+    "invert-kv": "^1.0.0",
+    "is-accessor-descriptor": "^0.1.6",
+    "is-arrayish": "^0.2.1",
+    "is-binary-path": "^1.0.1",
+    "is-buffer": "^1.1.6",
+    "is-core-module": "^2.3.0",
+    "is-descriptor": "^0.1.6",
+    "is-data-descriptor": "^0.1.4",
+    "is-extendable": "^0.1.1",
+    "json-server": "^0.16.3"
```

Comment:
If you are already including this as a project level dependency, then maybe also create a run script to start the server?  
Same goes for `http-server` 😃